common/src/main/java/org/apache/falcon/security/SecurityUtil.java

```diff
@@ -115,6 +115,16 @@ public static boolean isAuthorizationEnabled() {
                 "falcon.security.authorization.enabled", "false"));
     }
 
+    /**
+     * Checks if CSRF filter is enabled in the configuration.
+     *
+     * @return true if falcon.security.csrf.enabled is enabled, false otherwise
+     */
+    public static boolean isCSRFFilterEnabled() {
```

Comment:
Good question. Somehow the current Hadoop CSRF filter RestCsrfPreventionFilter (from their source code) doesn't need kerberos authentication. It provides the standard header check. It also works with our current simple authentication option (i.e. pass user name in url).